spec/models/user_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  context 'associations for 1 user that created 2 posts' do
+    before :example do
+      @user = User.create!(first_name: 'so',
+                           last_name: 'so',
+                           email: 'so@gmail.com',
+                           password: '123456',
+                           password_confirmation: '123456')
+      ​
+      @post1 = Post.create!(content: 'This is the content of post 1',
+                            user_id: @user.id)
+      ​
+      @post2 = Post.create!(content: 'This is the content of post 2',
+                            user_id: @user.id)
+    end
```

Comment:
Consider using `let` instead of `before`.

Reply:
We stuck with using `before` because when we changed to let, our tests were unsuccessful.
But our tests are now working fine.